hack/community/readme.md

```diff
@@ -1,21 +1,49 @@
-name, description, and date. Once generated into the artifact directory, the 
-bundle is used in the windows-e2e-operator test. After it passes, the bundle can
-be used to release a new community WMCO.
+CI contains a COMMUNITY environment variable that defaults to false, and is only
+set to true when a community-4.x branch PR is opened. When true, CI is 
+signaled to copy the repo's bundle into the test artifact directory, edit the 
+bundle, and then use the revised bundle in the PR's aws-e2e-operator test. After
+the test passes, the bundle can be copied from the PR's artifact directory and 
+used to release a new community WMCO.
 
-### After the test passes
-- Fork and clone [community-operators-prod](https://github.com/redhat-openshift-ecosystem/community-operators-prod)
-repo.
-- Copy the community bundle from the ARTIFACT_DIR.
-- Paste the bundle contents into a new folder under 
-community-windows-machine-config-operator. The folder title should be the
-next version of the community WMCO that is to be released.
-- Scan over the bundle CSV and annotations to ensure all the necessary fields
-were replaced. Add or adjust fields as needed including olm.skipRange, name, 
-version, and replaces-mode.
+### Benefit
+Previously, a manual testing process was done to ensure that the changes to the
+Red Hat operators WMCO manifests were brought in while maintaining the
+differences needed by the community operator. This new process automates the
+testing portion of a community WMCO release by reducing the manual intervention
+done in the bundle and testing the bundle directly in CI.
 
-### Ask for reviews
-- Tag WINC team for review
+### Process
+1. Fork and clone [community-operators-prod](https://github.com/redhat-openshift-ecosystem/community-operators-prod)
+repo.
+2. Copy the manifests/ and metadata/ directories from the aws-e2e-operator test 
+artifact directory of the most recent community-4.x branch PR.
+3. Paste the bundle contents into a new folder under 
+operators/community-windows-machine-config-operator/<x.y.z>. The folder title 
+should be the next semantic version of the community WMCO that is to be 
+released.
+4. Scan over the CSV and annotation files to ensure all the necessary fields
+were replaced by the generate.sh script. The script handles replacing the date, 
+operator description, display name, and maturity.
+5. Manual changes needed in the CSV:
+   - Add 'community-' prefix to 
+     - 'metadata.name' field
+     - 'spec.install.spec.deployments.spec.template.spec.containers.env.value' 
+     field
+   - Add image path 
+   'quay.io/openshift-windows/community-windows-machine-config-operator:community-4.x-<PR-commit-hash>' 
+   to
+     - 'metadata.annotations.containerImage'
+     - 'spec.install.spec.deployments.spec.template.spec.containers.image'
+   - If this is a minor release:
+     - add
+       'spec.replaces: community-windows-machine-config-operator.v<x.y.z>' where 
```

Comment:
On minor versions `olm.skipRange` should be removed as well

Reply:
we don't do it on RH operator side and we haven't done it for all releases on community side, is that required?
https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/community-windows-machine-config-operator/5.1.1/manifests/community-windows-machine-config-operator.5.1.1.clusterserviceversion.yaml#L11

Reply:
[we used to do it](https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/community-windows-machine-config-operator/3.1.1/manifests/community-windows-machine-config-operator.3.1.1.clusterserviceversion.yaml) and I think we should be doing it moving forward. Its not 'required' for things to work, but it doesn't make sense when considering WMCO's upgrade graph, and makes it unclear as to where WMCO will be upgraded to from a previous major version